integration/default_set.yaml

```diff
@@ -277,3 +277,20 @@ tests:
       - command: -N import syslog
         input: data/syslog/syslog.log
       - command: -N export ascii '#type == /syslog.*/'
+  Explore Zeek:
+    tags: [explore, zeek]
+    fixture: ServerTester
+    steps:
+      - command: import zeek
+        input: data/zeek/conn.log.gz
+      - command: explore --after=3min 'uid == "Yaix3QBP3Xg"'
+  Explore Zeek With Overlap:
+    tags: [explore, zeek]
+    # This is a stress test for the deduplication: The query selects every
+    # event, and the huge timebox ensures that every event is included in
+    # the context of every other event.
+    steps:
+      - command: -N import zeek
+        input: data/zeek/conn.log.gz
+      - command: -N explore --before=10000h --after=10000h '#type == /zeek.*/'
+        transformation: wc -l
```

Comment:
Side comment: This test actually gets hit pretty hard by our current slow json/zeek(?) export bug, taking almost 30 seconds to write 8462 json results on my machine.

Reply:
Can you add that as a comment into the story?